app/(platformMap)/layout.tsx

```diff
@@ -16,8 +16,13 @@ export default function Layout({
   bottom: React.ReactNode
   belowMap: React.ReactNode
 }) {
-  const [ref, { height }] = useMeasure<HTMLDivElement>()
   const path = usePathname()
+  const params: { regionId?: string } = useParams()
+  let [ref, { height }] = useMeasure<HTMLDivElement>()
+
+  if (height < 420) {
+    height = 420
```

Comment:
Where did you get this value for a minimum height?

Reply:
Anything less than 420 cuts off buoys on the map with the current Maine-centered zoom.